### ISSUE TITLE
Inconsistent Page Scroll Position When Navigating Between Pages

### ISSUE THREAD

**Issue by KriteshTimsina** [leapfrogtechnology/frogtoberfest#91]
**Issue:**
The current implementation of page navigation from the Home page to the Checker page does not reset the scroll position to the top of the page on arrival, resulting in a disorienting user experience.

**Description:**
- **Steps to Reproduce:**
  1. Access the website on any device.
  2. Navigate to a specific section within a page (e.g., a section on the Home page).
  3. Click on another route/page (e.g., navigate to the Checker page).
  4. Observe the scroll position on the new page.

**Expected Behavior:**
- When navigating between pages, the scroll position should reset to the top of the page, providing users with a consistent starting point.

**Actual Behavior:**
- The website retains the scroll position when moving between pages, which may lead to users starting at an unexpected point in the content.

**Suggested Solution:**
- Implement a mechanism to reset the scroll position to the top of the page when a user navigates to a new route/page.

**Screenshots/Code Snippets:**
https://github.com/leapfrogtechnology/frogtoberfest/assets/103115789/407d12e3-307c-4893-a468-f236aad8fa55

**Comment by luckyklyist** [comment]
@KriteshTimsina , when clicking on the 'Participation' link, you are redirected to the correct path on the homepage, which is 'https://frogtoberfest.lftechnology.com/#participation.' Therefore, I don't see any errors here. The same applies to other navigation items

**Comment by KriteshTimsina** [comment]
You're correct. All I'm saying is participation pages has 3 sections. Clicking on each link navigates to the respective section but doesn't close the sidebar.

**Comment by luckyklyist** [comment]
Which sidebar do you mean on the participation page?

**Comment by KriteshTimsina** [comment]
I'm referring to sidebar on small screens. I've attached the recording on the issue above.

**Comment by luckyklyist** [comment]
that's on the issue no : #90

**Comment by KriteshTimsina** [comment]
My apologies! I was actually referring to issue #90 all along. This issue pertains to scrolling to the top of the page. Please try scrolling to the bottom of the participation page and then navigate to the checker page. The page doesn't load from the top; instead, it loads from the previous position.
